docs/guides/node-operator-manual.md

```diff
@@ -276,3 +326,35 @@ docker run --rm -it \
   --datadir /root/.lighthouse/data \
   --directory /root/validator_keys
 ```
+
+### Checking the keys of all Lido Node Operators
+
+After keys are submitted, it's encouraged to wait for the transactions to be mined and to check how they appear on the chain.
+
+#### Lido CLI
+
+Make sure Python with pip is installed and then run:
+
+```sh
+pip install lido-cli
+lido-cli --rpc http://1.2.3.4:8545 validate_network_keys
+```
+
+This operation checks every Lido key for validity and thus is very CPU-intensive and takes a long time to process the keys.
+
+You would need an RPC endpoint - a local node / RPC provider (eg Alchemy/Infura).
+
+If the new keys are present and valid, Node Operators can vote for increasing the key limit for the Node Operator.
```

Comment:
Need a note on what to do if there's something wrong and keys aren't valid

Reply:
Agreed, adding this